tests/test_config.py

```diff
@@ -56,3 +56,25 @@ def test_configs_are_loaded_with_backwards_compatibility(client):
     assert sl.config['login_url'] == '/custom_login/'
     assert sl.config['logout_url'] == '/custom_logout/'
     assert sl.config['home_url'] == '/custom_home/'
+
+def test_messages_disabled(app):
+    app = Flask(__name__)
+    app.config['SECRET_KEY'] = 'secret-here'
+    sl = SimpleLogin(app, messages=False)
+    for message in sl.messages.values():
+        assert not message.enabled
+
+def test_messages_customized(app):
+    app = Flask(__name__)
+    app.config['SECRET_KEY'] = 'secret-here'
+    custom_message_dict = {
+        'login_success': Message('login_success_custom_message', category='login_success_custom_category'),
+        'logout':Message(enabled=False)
```

Comment:
Minor code style edit:

```suggestion
        'logout': Message(enabled=False)
```